python/ray/node.py

```diff
@@ -208,6 +208,9 @@ def __init__(self,
 
         if head:
             ray_params.update_if_absent(num_redis_shards=1)
+            gcs_server_port = os.environ.get("RAY_GCS_SERVER_PORT")
+            if gcs_server_port:
+                ray_params.update_if_absent(gcs_server_port=gcs_server_port)
```

Comment:
does it make sense to define this in constants.py or something?

Reply:
Done!